experimental/maven-plugin/README.md

```diff
@@ -11,9 +11,9 @@ The plugin wraps the [the slsa verifier](https://github.com/slsa-framework/slsa-
 
 ## Prerequisites
 
-To use the plugin you must have Go, Java and Maven installed. It has currently only been tested on Ubuntu.
```

Comment:
@AdamKorcz please confirm this change is correct

Reply:
It is correct.